src/main/java/org/opensearch/knn/index/codec/jvector/JVectorRandomAccessWriter.java

```diff
@@ -26,7 +25,10 @@ public JVectorRandomAccessWriter(IndexOutput indexOutputDelegate) {
 
     @Override
     public void seek(long position) throws IOException {
-        log.info("Pretending to be seeking to position {} in JVectorRandomAccessWriter, in practice it is not seeking anywhere.", position);
+        log.warn("Pretending to be seeking to position {} in JVectorRandomAccessWriter, in practice it is not seeking anywhere.", position);
+        throw new UnsupportedOperationException(
```

Comment:
Going forward all seeks should now throw an exception and will fail everything. We will no longer pretend that we are seeking on the writer and go back to business as usual because it breaks immutability and fail things in all kinds of ways.